### ISSUE TITLE
Excel User Guide improvements

### ISSUE THREAD

**Issue by ggrammel** [Telecominfraproject/oopt-gnpy#95]
2018-07-21

Excel_userguide.rst:
- Line 14: The file meshTopologyExampleV2 contains also a tab “service” while the text only says:
(In progress) The File MAY contain an additional sheet:

- Eqt

- Line 26: Questions about the following text:
**City** is used for the name of a node of the digraph. It accepts letters, numbers,commas,underscore,dash, blank... (not exhaustive).
    - often CSV is used as format. Suggestion is to disallow a comma for any entry in the table
    - should it be called “diagram” other than “digraph”?
- line 54: unclear what this means:
'z to a' direction MUST NOT be repeated in a different line.
- line 33: typo s/fuesed/fused
- line 79: should explain why, e.g.: “if amplifier sites are unknown, spans need to be broken up. The current assumption is that links larger than 150km will require intermediate amplification. This assumption will be revisited when Raman amplification is added”
- line 105: add “… and shall be unique”.
- line 116: appears to me that only 12 columns are used
- line 121: The text says:
If the sheet is present, it MUST have as many lines as egress directions of ROADMs defined in Links Sheet. 
    - the later example contains more lines (ILA to ILA connections). does it mean those ILA2ILA lines are optional?
- line 146: How does the code help?  A little more description is useful:
    - `create_eqpt_sheet.py <examples/create_eqpt_sheet.py>`_ helps you to create a template for the mandatory entries of the list.
- Question to line 193: Since TRX types are mandatory input parameters, they have to be manually pre-selected. Would be good to be able to put here a “most-sensitive” wildcard, which would provide the highest modulation scheme supporting the service. This may require that the list of TXP is sequenced, i.e. it may be preferable to use a 400G TXP with 1dBm output power over one with 4dBm output power.
- line 239: an example and the interpretation of the output would help to interpret this output:
Computing path from trx Brest_KLA to trx Vannes_KBE
with explicit path: ['Lannion_CAS', 'Lorient_KMA', 'trx Vannes_KBE']
pi -8.0
pi -5.0
pi -17.0
- line 246: the description is unclear
Important note: path_requests_run.py is not a dimensionning tool, it only computes path feasibility assuming full load with system parameters input in the service file. 
    - we do not have a definition of “dimensioning tool”. Some additional text may help.
    - what means “full load” as the service file describes a partial load (i.e. the services) in the network?
The network topology is created only once for the set of requests. 
    - which “topology” is meant? the fiber topology should be clear from the beginning. I believe it means the “wavelength routes”.
As a result the transceiver element acts as a "logical starting/stopping point" for the spectral information propagation. At that point it is not meant to represent the capacity of add drop ports
    - what does it mean to say “not meant to represent the capacity …”? Perhaps an issue because of the PCE model, but it confused me as there is no precedent where capacity is mentioned.
As a result transponder type is not part of the network info. it is related to the list of services requests.
    - I would rather call it “connectivity demand” as “services” may require more information.

**Comment by dutc** [comment]
@EstherLerouzic, could you look into some of these questions about the Excel user guide? Thanks!

**Comment by EstherLerouzic** [comment]
Hello @ggrammel ,
This pull request #102 answers most of your questions and suggestions.
Your suggestion "Question to line 193: Since TRX types are mandatory input parameters, they have to be manually pre-selected. Would be good to be able to put here a “most-sensitive” wildcard, which would provide the highest modulation scheme supporting the service. This may require that the list of TXP is sequenced, i.e. it may be preferable to use a 400G TXP with 1dBm output power over one with 4dBm output power." is indeed a feature that I plan to include.

definition of “dimensioning tool" : hope that my comments are clearer now...
This is the work I am starting now: adding some "resource " occupation to be able to count network resource occupation. Let's talk of that on our next call !
regards
Esther

**Comment by ggrammel** [comment]
we are good and can close the issue! Thanks for updating
